.buildkite/test.sh

```diff
@@ -3,4 +3,4 @@
 set -ex
 cd $(dirname "${BASH_SOURCE[0]}")
 
-./yarn-run.sh "test --verbose"
+./yarn-run.sh "test" "coverage"
```

Comment:
I feel like there's a story here.

```suggestion
./yarn-run.sh test coverage
```